media.css

```diff
@@ -0,0 +1,1400 @@
+  }
+
+  .hidden-1024 {
+    display: flex;
+    align-items: center;
+  }
+
+  .stories-item__date {
+    font-size: 15px;
+    line-height: 14px;
+    font-weight: bold;
+    color: #d1d1d1;
+    opacity: 1;
+    text-align: right;
+  }
+
+  .gallery {
+    display: block;
+    height: 17%;
+    background: linear-gradient(2.6deg, #fff 50%, #000 51%);
+    margin-bottom: -5px;
+    width: 100%;
+  }
+
+  .gallery__title {
+    font-size: 25px;
+    line-height: 24px;
+    color: #e8e8e8;
+    opacity: 1;
+    text-align: center;
+    text-transform: uppercase;
+    font-weight: bold;
+    margin-bottom: 17px;
+    margin-right: 0;
+  }
+
+  .gallery-content {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: 3;
+    margin-left: 48px;
+    margin-right: 48px;
+  }
+
+  .gallery-content__item {
+    display: block;
+    margin-bottom: -4px;
+  }
+
+  .gallery__photo {
+    width: 100%;
+  }
+
+  .hidden-1024 {
+    display: block;
+  }
+
+  .footer {
+    width: 100%;
+  }
+
+  .footer__content {
+    max-width: 75%;
+    padding-top: 40px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .footer__column {
+    text-align: center;
+    margin-bottom: 52px;
+  }
+
+  .footer__column-title,
+  .footer-column__nav-link {
+    font-size: 15px;
+    line-height: 14px;
+  }
+
+  .copyright {
+    padding-top: 30px;
+    margin-top: -52px;
+  }
+
+  .footer__button {
+    width: 150px;
+    height: 37px;
+    margin-top: 29px;
+    margin-bottom: 104px;
+    margin-right: auto;
+    margin-left: auto;
+  }
+
+  .social-icon {
+    width: 100%;
+  }
+
+  .hidden-768,
+  .hidden-375 {
+    display: none;
+  }
+}
+
+@media (max-width: 319px) {
+  .wrapper {
+    max-width: 319;
+    margin: 0 auto;
+  }
+
+  .header {
+    grid-template-columns: 1fr 1fr;
+    margin-bottom: 26px;
+  }
+
+  .header__menu,
+  .header__search,
+  .header__user-block,
+  .header__cart,
+  .header__language-block {
+    display: none;
+  }
+
+  .header__logo-image {
+    display: none;
+  }
+
+  .header__logo-image_adaptive {
+    width: 104px;
+    height: 22px;
+    display: flex;
+    align-items: center;
+    margin-left: 20px;
+  }
+
+  .header__menu-bar {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
+
+  .menu-bar {
+    display: block;
+    width: 41px;
+    height: 17px;
+    margin-right: 20px;
+  }
+
+  .top-content {
+    background-image: url(img/header-img-375.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    height: 191px;
+  }
+
+  .top-content__block {
+    /* width: 355px;
+    height: 169px; */
+    padding-left: 20px;
+  }
+
+  .top-content__h1 {
+    font-size: 20px;
+    line-height: 19px;
+    margin-bottom: 9px;
+  }
+
+  .link {
+    font-size: 12px;
+    line-height: 11px;
+    color: #fff;
+  }
+
+  .arrow {
+  	width: 40px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)